src/main/readout.go

```diff
@@ -33,7 +33,7 @@ import (
 	"math"
 	"os"
 	"path/filepath"
-	"github.com/nDenerserve/SmartPi/src/smartpi"
```

Comment:
I have a fix for the makefile so you don't need to change this.  Should also help with the go get a bit.